tracer/src/Datadog.Tracer.Native/interop.cpp

```diff
@@ -216,6 +216,11 @@ EXTERN_C VOID STDAPICALLTYPE UpdateSettings(WCHAR* keys[], WCHAR* values[], int
     return trace::profiler->UpdateSettings(keys, values, length);
 }
 
+EXTERN_C VOID STDAPICALLTYPE GetIastMetrics(int* instrumentedSources, int* instrumentedPropagations, int* instrumentedSinks)
```

Comment:
As @andrewlock said, we should try to make this more future proof (Like defining something like this):
`
struct MetricsData
{
  int Id;
  int IntValue;
  string StringValue;
}
`

And defining the function like (returning the number of elements filled in the in/out array)
`
int GetMetrics(int maxItems, MetricsData* items);
`